packages/app/src/index.ts

```diff
@@ -0,0 +1,112 @@
+import { Context } from '@release-drafter/core'
+import { ApplicationFunction, Context as ProbotContext } from 'probot'
+import { ReleaseDrafterOctokit } from '@release-drafter/core'
+
+const app: ApplicationFunction = async (app, { getRouter }) => {
+	if (typeof getRouter === 'function') {
+		getRouter().get('/healthz', (_, response) => {
+			response.status(200).json({ status: 'pass' })
+		})
+	}
+
+	app.on(
+		[
+			'pull_request.opened',
+			'pull_request.reopened',
+			'pull_request.synchronize',
+			'pull_request.edited',
+			'pull_request_target.opened',
+			'pull_request_target.reopened',
+			'pull_request_target.synchronize',
+			'pull_request_target.edited',
+		] as never,
+		async (context: ProbotContext<'pull_request'>) => {
+			const pullRequest = context.pullRequest()
+
+			const myContext = new Context(
+				context.octokit as unknown as InstanceType<
+					typeof ReleaseDrafterOctokit
+				>,
+				{
+					...pullRequest,
+				},
+			)
+
+			const config = await myContext.config()
+		},
+	)
+
+	// const drafter = async (context) => {
```

Comment:
Commented code block, to be deleted?

Reply:
I am slowly implementing it. It is still in draft.